cmd/podman/main_local.go

```diff
@@ -50,12 +50,12 @@ func init() {
 	if err := rootCmd.PersistentFlags().MarkHidden("default-mounts-file"); err != nil {
 		logrus.Error("unable to mark default-mounts-file flag as hidden")
 	}
-	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.EventsBackend, "events-backend", "", "Events backend to use")
+	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.EventsBackend, "events-backend", "", `Events backend to use ("file"|"journal"|"none")`)
```

Comment:
are you sure the value is not `journald` instead of `journal`?

In docs/source/markdown/podman.1.md:
```
**--events-backend**=*type*

Backend to use for storing events. Allowed values are **file**, **journald**, and **none**.
```

Reply:
Nice catch! Indeed, `libpod/events/events.go` confirms that the actual value is `journald`. Fixed and pushed.

Many thanks for your keen eye. Have I mentioned (lately) how much I dislike manual duplication?...